grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -244,16 +245,36 @@ def testProgressTruncates20RowsTo10(self):
     row_number = 20
 
     test_table = _CreateDictTable(column_number, row_number)
-    test_table_json = json.Dump(test_table)
+    test_table_json = json.dumps(test_table)
 
     with osquery_test_lib.FakeOsqueryiOutput(stdout=test_table_json, stderr=""):
       flow_id = self._NewTestFlowStartAndRun(query)
       progress = flow_test_lib.GetFlowProgress(self.client_id, flow_id)
 
-    self.assertEqual(progress.total_rows_count, row_number)
+    self.assertEqual(progress.total_row_count, row_number)
     self.assertEqual(progress.partial_table.query, query)
     self._AssertTablesMatch(progress.partial_table, test_table[:10])
 
+  def testTotalRowCountIncludesAllChunks(self):
+    row_count = 100
+    split_pieces = 10
+
+    # Not using _CreateDictTable here in order to control the exact size of the
+    # test table in bytes, and thus control the number of chunks
+    cell_value = 'fixed'
+    table = [{'column1': cell_value} for _ in range(row_count)]
```

Comment:
This can be simplified to` [{'column1': cell_value}] * row_count` (it's fine, since it is used as immutable, constant value).